lib/classes/PluginManager.js

```diff
@@ -58,10 +55,10 @@ class PluginManager {
   }
 
   loadCorePlugins() {
-    const utils = new Utils();
     const pluginsDirectoryPath = path.join(__dirname, '../plugins');
 
-    const corePlugins = utils.readFileSync(path.join(pluginsDirectoryPath, 'Plugins.json')).plugins;
+    const corePlugins = this.serverless.utils
```

Comment:
Should be done without a dependency on the serverless instance (otherwise it's not encapsulated).